DigitalLearningSolutions.Data/Models/User/CentreAnswersData.cs

```diff
@@ -8,6 +8,7 @@ public class CentreAnswersData
         public CentreAnswersData(

```

Comment:
Rename this to `RegistrationFieldAnswers` and add a class `DelegateDetailsData`, and a constructor to turn `DelegateDetailsData` into `RegistrationFieldAnswers`.

Hopefully you can remove the existing constructor too.
I suggested that one should inherit the other, but that actually feels wrong, conceptually. Adding to the `DelegateDetailsData` model won't necessarily add to the registration fields we can use to generate groups.

(Can't comment on the class declaration 😢)